k8s/flink-operator/manifest/application.yaml

```diff
@@ -27,11 +27,10 @@ spec:
       url: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/flink-operator/README.md
     - description: Quick start with Flink Operator
       url: https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/developer_guide.md
-    - description: Apache Flink documentations
+    - description: Apache Flink documentation
       url: https://ci.apache.org/projects/flink/flink-docs-master/
     notes: |-
-      Follow these [examples](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/developer_guide.md#create-a-sample-flink-cluster)
-      to deploy your Flink jobs.
+      To deploy your Flink jobs, follow [these examples](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/developer_guide.md#create-a-sample-flink-cluster).
```

Comment:
Please update this to https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/user_guide.md#create-a-sample-flink-cluster.